content/doc/book/updating/windows.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: section
+title: Windows
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:left
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Windows
+
+There are two ways to upgrade Jenkins in Windows.
+One is via Windows MSI installer
+Other one is discussed later on the page.
+
+== Upgrading via MSI Installer
+you can download the latest LTS or Weekly MSI Installer to upgrade Jenkins.
+
+Just Download the Jenkins 
+link:https://www.jenkins.io/download/#downloading-jenkins[MSI Installer] from here and install the latest
+Jenkins LTS or Weekly.
+for more information visit this page.
+link:https://www.jenkins.io/doc/book/installing/windows/#windows-msi-installers[Installing Jenkins On Windows via MSI Installer]
+
+=== 1. Stop the running Jenkins service
+
+To stop Jenkins Please avoid shutting down the Java process or the Windows service. These are not usual commands. Use those only if your Jenkins is causing problems.
```

Comment:
```suggestion
To stop Jenkins please avoid shutting down the Java process or the Windows service. These are not usual commands.
Use those only if your Jenkins is causing problems.
```
This section does not seem very clear, please add additional context/text to help clarify the statement